entry/src/ohosTest/java/com/example/tanfujun/hintanimedittext/ExampleOhosTest.java

```diff
@@ -0,0 +1,56 @@
+package com.example.tanfujun.hintanimedittext;
```

Comment:
Applibgroup copyright can be added to this file.

Reply:
Done